coalib/results/Diff.py

```diff
@@ -354,6 +354,16 @@ def delete_line(self, line_nr):
         linediff.delete = True
         self._changes[line_nr] = linediff
 
+    def delete_lines(self, line_nr_start, line_nr_end):
+        """
+        Delete lines in a range, exclusive.
+        """
+        for line_nr in range(line_nr_start, line_nr_end):
+            # Flatten delete_lines to reduce computations
+            linediff = self._get_change(line_nr)
+            linediff.delete = True
+            self._changes[line_nr] = linediff
```

Comment:
hey, one of the reasons why our checks on GitHub are failing is that this function is untested, I think a simple doctest (simlar to around line ~340) will be neat and simple!